### PR TITLE
Add the first CHANGELOG.md

### PR DESCRIPTION
**Pull request by alexhung**
Per https://www.terraform.io/plugin/sdkv2/best-practices/versioning#changelog-specification

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## 2.13.0 (Feb, 1, 2022)
```
Comment (chb0github) [inline CHANGELOG.md:1]: mmm, this isn't your version

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+FEATURES:
+
+* **New Resources:** `artifactory_federated_x_repository` where `x` is one of the following [GH-296]:
```
Comment (chb0github) [inline CHANGELOG.md:5]: OH! You need a ci/cd check to make sure this changes with a release

File: CHANGELOG.md (hunk 1 above)
Comment (alexhung) [reply CHANGELOG.md:1]: What do you mean?

Comment (chb0github) [reply CHANGELOG.md:1]: what does `2.13.0` represent?

**Comment by martinm82** [comment]
> You need some process in place to make sure your changelog stays current

How about using conventional commits and using semantic-release? The one and only truth would be your commits and you would not need to change any changelogs anymore. All automated. 

If it is something you would be interested, I could open a PR.

**Comment by chb0github** [comment]
We'd love that! @martinm82

**Pull request merged**